python/ql/src/Statements/ReturnOrYieldOutsideOfFunction.ql

```diff
@@ -0,0 +1,25 @@
+/**
+ * @name return or yield are used outside of a function
+ * @description return and yield statements should be used only within a function.
```

Comment:
"return" should be upper case in both of these for consistency with the rest of the query suite.

Reply:
This is tricky one. When using Python keywords, we wrap them into single quotes, so it would be `'return'`. See example here https://help.semmle.com/wiki/display/PYTHON/%27apply%27+function+used with

> Name: 'apply' function used
> Description: The builtin function 'apply' is obsolete and should not be used.

Would having

>  * @name 'return' or 'yield' are used outside of a function
>  * @description 'return' and 'yield' statements should be used only within a function.

be acceptable based on what we are doing at other `.ql` queries metadata?

Reply:
Ah, looking at our suite of queries, I see that we're not really consistent about this in the first place (e.g. "`__init__ method returns a value`"). I think the single quotes help, but perhaps it would be better if it was rewritten to be of the form "syntax X causes behavior Y" (which is what @felicity-semmle usually suggests to me).